blueocean-dashboard/src/main/js/components/ColumnFilter.jsx

```diff
@@ -39,8 +43,14 @@ export class ColumnFilter extends Component {
         e.target.select();
     }
 
-    blur() {
+    blur = e => {
```

Comment:
Using arrow funcs for event handlers is the right way to go, but then we need to also update `render()` to set listers from `onEvent={e=>this.eventHandler(e)}` to `onEvent={this.eventHandler}` otherwise we don't get the benefits.